secrets.sh

```diff
@@ -438,11 +438,11 @@ EOF
 		if [[ $yml =~ ^=.*$ ]]; then
 		    yml="${yml/=/}"
 		fi
-		if [[ $yml =~ ^(.*/)?secrets(\.[^.]+)*\.yaml$ ]]
+		if [[ $yml =~ ^(.*/)?secrets([_.-][^.]+)*\.yaml$ ]]
```

Comment:
The only problem with this change is that it allowed us to have something like `secrets_-.yaml`. 
 AFAIR concepts, we always prefer to have a dir structure for different vars/secret files.

Reply:
I think as long as it's a valid file name and it does not cause problem downstream, you should leave it up to the user to decide. You should not be deciding on naming "style", all you need is a reasonable way of identifying secrets files. I vote to leave the fix as-is.